cmd/crank/init.go

```diff
@@ -0,0 +1,83 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/alecthomas/kong"
+	"github.com/go-git/go-billy/v5/osfs"
+	"github.com/go-git/go-git/v5"
+	"github.com/go-git/go-git/v5/storage/memory"
+
+	"github.com/crossplane/crossplane-runtime/pkg/errors"
+	"github.com/crossplane/crossplane-runtime/pkg/logging"
+)
+
+// initCmd initializes a repository from a template
+type initCmd struct {
+	Function initFunctionCmd `cmd:"" help:"Initialize a Function from the repo template for a selected language."`
+}
+
+type initFunctionCmd struct {
+	Name string `arg:"" help:"Name of the Function to initialize."`
+
+	Language  functionLanguage `short:"l" help:"Language of the Function to initialize." enum:"go" default:"go"`
+	Directory string           `short:"d" help:"Path of the directory to initialize." default:"." type:"path"`
+}
+
+type functionLanguage string
+
+const (
+	functionLanguageGo functionLanguage = "go"
+)
+
+func (c *initFunctionCmd) GetURLForLanguage() (string, error) {
+	switch c.Language {
+	case functionLanguageGo:
+		return "https://github.com/crossplane/function-sdk-go", nil
```

Comment:
this should be https://github.com/crossplane/function-template-go instead of the SDK repo, right?

Reply:
😅 you are right